java/client/src/main/java/glide/api/commands/GenericBaseCommands.java

```diff
@@ -542,4 +547,46 @@ CompletableFuture<Boolean> pexpireAt(
      * }</pre>
      */
     CompletableFuture<Long> touch(String[] keys);
+
+    /**
+     * Copies the value stored at the <code>source</code> to the <code>destination</code> key.
```

Comment:
Document that this operation will fail when destination key exists (unless `replace == true`).